test/Generators/Microsoft.Gen.Logging/TestClasses/MessageTestExtensions.cs

```diff
@@ -2,46 +2,38 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using Microsoft.Extensions.Logging;
-using Microsoft.Extensions.Telemetry.Logging;
 
 namespace TestClasses
 {
     internal static partial class MessageTestExtensions
     {
-        [LogMethod(0, LogLevel.Trace, null!)]
+        [LoggerMessage(0, LogLevel.Trace, null!)]
         public static partial void M0(ILogger logger);
 
-        [LogMethod(1, LogLevel.Debug, "")]
+        [LoggerMessage(1, LogLevel.Debug, "")]
         public static partial void M1(ILogger logger);
 
-        [LogMethod(2, LogLevel.Debug)]
+#pragma warning disable LOGGEN030
```

Comment:
Just curious, what is this warning about?

Reply:
"nothing is being logged by this log method". If there is no message and no parameters to the logging method.